src/extension/features/budget/pacing/index.js

```diff
@@ -1,15 +1,17 @@
 import debounce from 'debounce';
 import { Feature } from 'toolkit/extension/features/feature';
 import { isCurrentMonthSelected } from 'toolkit/extension/utils/ynab';
-import { getEmberView } from 'toolkit/extension/utils/ember';
 import {
   getDeemphasizedCategories,
   pacingForCategory,
   setDeemphasizedCategories,
 } from 'toolkit/extension/utils/pacing';
 import { formatCurrency } from 'toolkit/extension/utils/currency';
 import { l10n } from 'toolkit/extension/utils/toolkit';
-import { budgetRowInChangesSet } from 'toolkit/extension/features/budget/utils';
+import {
+  budgetRowInChangesSet,
```

Comment:
@joshmadewell, it appears this function was [removed from the utils](https://github.com/toolkit-for-ynab/toolkit-for-ynab/pull/3428/files#diff-6d255056c617097c5f83583df5ae4eb245982efff34b3b753b64d3b2529697bdL42), but is still being imported on this line and used on line 61. I noticed it when I ran `yarn build:development` in the repo:

```
WARNING in ./src/extension/features/budget/pacing/index.js 48:8-29
export 'budgetRowInChangesSet' (imported as 'budgetRowInChangesSet') was not found in 'toolkit/extension/features/budget/utils' (possible exports: GOAL_TABLE_CELL_CLASSNAME, ensureGoalColumn, getBudgetMonthDisplayMasterCategory, getBudgetMonthDisplaySubCategory)
 @ ./src/extension/features/index.ts 73:0-41 301:5-11 474:4-10
 @ ./src/extension/ynab-toolkit.tsx 2:0-54 98:20-36
 @ ./src/extension/index.js 1:0-45 2:22-33
```

Was that function intentionally removed and thus it should stop being used here? Or should the function be restored so that it _can_ be used here?

Reply:
Should [Pacing's observe method](https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/e8a0bf2b132f2ee7f3d87cecb1931de5d4da6a3b/src/extension/features/budget/pacing/index.js#L58-L64) be updated to match the updated [DisplayTargetGoalAmount's observe method](https://github.com/toolkit-for-ynab/toolkit-for-ynab/pull/3428/files#diff-25c518e08cb46b4b8a8ac7f9dc41c8a4ce0c92ef6584e9f1f7c49953078d4c32R33-R35)?